### PR TITLE
Update @hapi/joi references to joi

### PR DESCRIPTION
**Pull request by jonathansamines**

### CODE REVIEW

File: static/lib/tutorials/en_US/expresstohapi.md [hunk 1]
```diff
@@ -545,7 +545,7 @@ To use bell, simply register the plugin and configure the strategy with `server.
 
 ## <a name="joi" ></a> express-validator -> joi
 
-To validate data in Express, you make use of the `express-validator` plugin. One of the biggest drawbacks to `express-validator` is that while you can validate a request, there is no clear way of validating a response. In hapi, you use the [joi](https://github.com/sideway/joi) module, which can validate requests and responses with ease. Joi allows you to create your own validations with a simple and clean object syntax. For a more in-depth look at validation in hapi, please see the [validation](/tutorials/validation/?lang=en_US) tutorial.
+To validate data in Express, you make use of the `express-validator` plugin. One of the biggest drawbacks to `express-validator` is that while you can validate a request, there is no clear way of validating a response. In hapi, you use the [joi](https://joi.dev) module, which can validate requests and responses with ease. Joi allows you to create your own validations with a simple and clean object syntax. For a more in-depth look at validation in hapi, please see the [validation](/tutorials/validation/?lang=en_US) tutorial.
```
Comment (Nargonath) [inline static/lib/tutorials/en_US/expresstohapi.md:548]: I'm not sure we want to change the target of the link.

**Review by Nargonath — COMMENTED** [review]
Good catch on the `@hapi/joi` in the tutorials

**Comment by lloydbenson** [comment]
I think moving to joi.dev is fine. Initially we changed some links to just the github module because joi.dev didn't exist yet.

**Review by lloydbenson — APPROVED** [review]

**Comment by Nargonath** [comment]
Alright I didn't remember the timeline.

**Review by Nargonath — APPROVED** [review]

**Pull request merged**